From 7c505c4e60fb451b4e1f70991bac126902eefe87 Mon Sep 17 00:00:00 2001
From: Ilkka Koskinen <ilkka.koskinen@intel.com>
Date: Tue, 14 Oct 2014 16:29:48 -0700
Subject: [PATCH] acpi thermal: Clear the error value of last
 acpi_bus_get_device()

Previously the return value of the last acpi_bus_get_device()
was returned. However, since we only report those issues,
it should be cleared to continue as expected.

Change-Id: Ib0c63c1ad3ab88896f15563a33f2ca7c310f4ede
Signed-off-by: Ilkka Koskinen <ilkka.koskinen@intel.com>
---
 drivers/acpi/acpi_thermal_rel.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/acpi/acpi_thermal_rel.c b/drivers/acpi/acpi_thermal_rel.c
index 219a64f..9bf7785 100644
--- a/drivers/acpi/acpi_thermal_rel.c
+++ b/drivers/acpi/acpi_thermal_rel.c
@@ -131,6 +131,8 @@ int acpi_parse_trt(acpi_handle handle, int *trt_count, struct trt **trtp,
 			pr_warn("Failed to get target ACPI device\n");
 	}
 
+	result = 0;
+
 	*trtp = trts;
 	/* don't count bad entries */
 	*trt_count -= nr_bad_entries;
-- 
1.9.1

